packages/create-gatsby/src/index.ts

```diff
@@ -11,6 +11,11 @@ import { plugin } from "./components/plugin"
 import { makePluginConfigQuestions } from "./plugin-options-form"
 import { center, rule, wrap } from "./components/utils"
 import { stripIndent } from "common-tags"
+import { trackCli } from "gatsby-telemetry"
+import crypto from "crypto"
+
+const sha256 = (str: string): string =>
+  crypto.createHash(`sha256`).update(str).digest(`base64`)
```

Comment:
```suggestion
  crypto.createHash(`sha256`).update(str).digest(`hex`)
```

Just verified and we use the hex encoding elsewhere, so let's use it for consistency. Maybe we should expose this from telemetry but to avoid diving to the rabbithole let's go with this